asposeimagingcloud/models/requests/create_image_frame_range_request.py

```diff
@@ -0,0 +1,173 @@
+#  coding: utf-8
+#  ----------------------------------------------------------------------------
+#  <copyright company="Aspose" file="create_image_frame_range_request.py">
+#    Copyright (c) 2018-2019 Aspose Pty Ltd. All rights reserved.
```

Comment:
Please, change the copyright period to 2020 in all the new classes

Reply:
I've updated the copyright dates.